pinot-common/src/main/java/org/apache/pinot/common/function/FunctionRegistry.java

```diff
@@ -38,19 +42,14 @@
 
 /**
  * Registry for scalar functions.
- * <p>TODO: Merge FunctionRegistry and FunctionDefinitionRegistry to provide one single registry for all functions.
  */
 public class FunctionRegistry {
```

Comment:
Note to reviewer:  this class will be maintain until
1. all Transform/Scalar functions are unified
2. all legacy, arg-count based function resolution logics are deprecated